internal/receiver/discoveryreceiver/README.md

```diff
@@ -347,6 +347,8 @@ In this case `some.field.with.periods` can be referenced via:
 expr: 'ExprEnv["some.field.with.periods"] contains "value"'
 ```
 
+Note: `typeOf` should be used instead of `type` for matching types.
+
```

Comment:
These are different rules than the receiver creator so don't have an existing `type`

```suggestion
```

Reply:
ah, thanks, I thought this receiver had similar type function